edx-platform/pearson-certiport-theme/lms/templates/header/navbar-logo-header.html

```diff
@@ -0,0 +1,42 @@
+## mako
```

Comment:
Why are you adding this module in certiport and not in certprep?

Reply:
@anfbermudezme this module was removed in CertPREP, so I can do the same with this theme.

Reply:
What was that reason?

Reply:
@anfbermudezme  It didn't contain any custom change.